### PR TITLE
Use regex parser

### PR DESCRIPTION
**Pull request by Siphonophora**

### CODE REVIEW

**Comment by Siphonophora** [comment]
Just prepping release notes.

Release notes: 

* Update core testlogger to 3.0.78 for xunit test adapter
* Fix: Explicit tests should be marked as Skipped. See
  https://github.com/spekt/nunit.testlogger/issues/86
* Replace Test Case name parser **Possible Breaking Change**
    * For most or maybe all users the new parser should fix the issues shown below, without introducing new issues. In case you do encounter any new parsing failures a feature flag `Parser=Legacy` has been added to use the prior parser. See [logger config wiki](https://github.com/spekt/testlogger/wiki/Logger-Configuration) for details. 
    * Fix: Test case parse error if name contains special characters. See
  https://github.com/spekt/nunit.testlogger/issues/90
    * Fix: Covers several parsing issues. Thanks @becha2 for all the detailed examples.
       https://github.com/spekt/testlogger/issues/28
    * Fix: Log member data. Thanks @BottlecapDave for the issue report and @hach-que for the draft fix. 
      https://github.com/spekt/junit.testlogger/issues/50
    * Fix: Issue parsing chars. Thanks @binarycow for the issue report.
       https://github.com/spekt/nunit.testlogger/issues/90
    * Reduce log verbosity: The parser, if it encounters problems, will only output one warning per run to the console instead of one per problem

**Comment by Siphonophora** [comment]
@codito  this is ready

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Changelog
 
-## Unreleased (v3.1.x)
+## All versions staring in 2022
+A changelog is maintained on the releases page of the [XUnit Test Logger GitHub repository](https://github.com/spekt/xunit.testlogger/).
```
Comment (codito) [inline CHANGELOG.md:4]: @Siphonophora there's one challenge to this: if we do infrequent releases, e.g. every quarter or so, it becomes a hard task to go through all changes and collate those :( Changelog.md solves this in a rudimentary way. Curious what are the challenges with changelog.md?

Comment (Siphonophora) [reply CHANGELOG.md:4]: Oh, do you accumulate items under the unreleased tag as you go? That makes sense. I was planning to use the release page b/c we don't know the release version number until after it's built.

I'm happy to do that here.

Comment (codito) [reply CHANGELOG.md:4]: Yes, we keep collecting the changes incrementally :)

Comment (Siphonophora) [reply CHANGELOG.md]: All set

**Pull request merged**